include/NovelRT/NovelRunner.h

```diff
@@ -23,7 +23,7 @@ namespace NovelRT {
      * Use this to define game behaviour that is required to be done per-frame.
      */
     Utilities::Event<Timing::Timestamp> Update;
-
+    
```

Comment:
What happened here exactly?

Reply:
_Nothing to see here...just an extra space is all 👍 _